cpg-core/src/test/kotlin/de/fraunhofer/aisec/cpg/enhancements/EOGTest.kt

```diff
@@ -321,21 +321,21 @@ internal class EOGTest : BaseTest() {
                 en = Util.Edge.EXITS,
                 n = prints[0],
                 cr = Connect.SUBTREE,
-                refs = listOf(fs.initializerStatement)
+                refs = listOf(fs.initializerStatement!!)
             )
         )
         assertTrue(
             Util.eogConnect(
                 en = Util.Edge.EXITS,
-                n = fs.initializerStatement,
+                n = fs.initializerStatement!!,
                 cr = Connect.SUBTREE,
-                refs = listOf(fs.condition)
+                refs = listOf(fs.condition!!)
             )
         )
         assertTrue(
             Util.eogConnect(
                 en = Util.Edge.EXITS,
-                n = fs.condition,
+                n = fs.condition!!,
```

Comment:
is the compiler asking for them now? They were not necessary when I added the changes to the function

Reply:
The problem was that `condition` was always nullable, however this was never explicitly modelled as such in the Java class. Now after converting to Kotlin it is definitely a nullable `Node?` and now Kotlin complains, since eogConnect wants a `Node`. I was playing around with making arguments of the function to accept `Node?` but I somehow lost my nerve :D

Reply:
I may have a solution that now does not have any `!!`